Makefile

```diff
@@ -154,16 +159,15 @@ endif
 .PHONY: clean
 clean:
 	@if [ -d deps/jerryscript ]; then \
-		make -C $(JERRY_BASE) -f targets/zephyr/Makefile.zephyr clean; \
-		make -C $(JERRY_BASE) -f targets/zephyr/Makefile clean; \
+		make -C $(JERRY_BASE) -f targets/zephyr/Makefile clean BOARD=$(BOARD); \
```

Comment:
This seems a bit odd. I usually wouldn't pass a BOARD to make clean, so this will probably only clean a101?